docs/groovyscript/mods/thaumcraft/Research.md

```diff
@@ -0,0 +1,177 @@
+# Research
+
+### Thaumometer Scanning
+
+The following methods allow for scanning a specified Object with a thaumometer to discover the Object's component aspects.
+
+```groovy
+mods.thaumcraft.Research.addScannable(Block)
+mods.thaumcraft.Research.addScannable(Enchantment)
+mods.thaumcraft.Research.addScannable(Material)
```

Comment:
what is a "Material"? the others are all understandable (note: we should definitely get bracket handlers for them)